packages/pds-ember/app/styles/pds/components/link/_config.scss

```diff
@@ -1,75 +0,0 @@
-$_module: "PDS.Components.Link";
```

Comment:
File has been refactored:

* extract default link styles into `link/default/__config.scss`
* rename as `link/__config.scss` to communicate that module contents are meant to be used internally (contents can change at any time)